docs/access-token.md

```diff
@@ -0,0 +1,92 @@
+# Access Token
+
+If you want to automate tasks with the Argo Server API or CLI, you will need an access token. 
+
+Firstly, create a role with minimal permissions. This example role for jenkins only permission to update and list workflows:
+
+```yaml
```

Comment:
People want to do less things, so I think giving an easy option (or both) of `kubectl create role jenkins-role --verb=list,update --resource=orkflows.argoproj.io` might be better - don't need to copy, paste and save the yaml file.

Reply:
I'm so used to GitOps :(